package.json

```diff
@@ -59,12 +59,13 @@
     "view-cover": "istanbul report html && opn ./coverage/index.html",
     "browser": "run-browser test/index.js",
     "phantom": "run-browser test/index.js -b | tap-spec",
-    "dist": "browserify --standalone virtual-dom index.js > dist/virtual-dom.js",
+    "dist": "mkdir -p dist; browserify --standalone virtual-dom index.js > dist/virtual-dom.js",
```

Comment:
shouldn't we be using a package rather than relying on the OS - e.g. windows doesn't have `mkdir`?